db/migrate/20150421191213_add_column_to_company.rb

```diff
@@ -0,0 +1,6 @@
+class AddColumnToCompany < ActiveRecord::Migration
+  def change
+    add_column :companies, :hiring, :boolean
```

Comment:
I agree with Chris, that this might be nice to have a status for an admin to keep track of the communications with the companies but I think we can wait to add that in until the future weeks when we work on those features. That way they can filter by just the companies that haven't been contacted yet.

This looks good to me for now though!